corehq/messaging/util.py

```diff
@@ -52,14 +52,16 @@ def rule_initiation_key_minutes_remaining(self):
     def set_initial_progress(self, shard_count=0):
         # shard_count is passed when tasks are run per each shard
         self.client.set(self.current_key, 0)
-        self.client.set(self.total_key, 0)
```

Comment:
this might still be good to have to "set things up"

Reply:
I am actually setting `total_key` before this is called, so the total key is already set up with a value.